ui/run.sh

```diff
@@ -149,6 +152,8 @@ case "$(basename ${0})" in
         o "Pinging Management API Endpoint until ready"
         run ecsconfig ping -c -x || exit $?
         run ecsconfig sp -a || exit $?
+	o "Sleeping for 15 minutes for Storage Pool to be ready"
```

Comment:
Use the correct API calls to determine cluster readiness, not sleep.

Reply:
We need to check and implement API call , will take more time , something can work on 3.6.2.